### ISSUE TITLE
What's happening with seeding?

### ISSUE THREAD

**Issue by Naatan** [sharkone/BitTorrent.bundle#37]
I see no settings about seeding, does this basically run an indefinite seed for each torrent on my machine? 

It'd be nice if there was a setting for this or at least some info on what is happening.

**Comment by sharkone** [comment]
When watching a movie, it's being downloaded/uploaded in the background. As long as you are actively watching the movie, uploading continues. After some period of inactivity (30s by default), the torrent will be paused (download and upload will stop), but kept, just in case you just went away to grab a beer or something :)
After a longer period of time (10min by default), the torrent will be removed and the Keep Files option is not checked in the Preferences, the downloaded files will also be deleted.

So basically, you're only seeding while watching.
